app/services/questions_service.rb

```diff
@@ -76,13 +76,18 @@ def delegated_functions_on_any_proceeding(_proceeding)
     @proceedings.any? { |proceeding| proceeding[:delegated_functions_used].to_s == "true" }
   end
 
+  def delegated_functions_and_domestic_abuse_with_non_applicant(_proceeding)
+    @proceedings.any? { |p| p[:ccms_code].match(DOMESTIC_ABUSE_CODE_REGEXP) && p[:client_involvement_type] != "A" && p[:delegated_functions_used].to_s == "true" }
+  end
+
   def defendant_on_this_proceeding(proceeding)
     match = @proceedings.find { |e| e[:ccms_code] == proceeding.ccms_code && e[:client_involvement_type] == "D" }
     match.present?
   end
 
-  def delegated_functions_and_domestic_abuse_with_non_applicant(_proceeding)
-    @proceedings.any? { |p| p[:ccms_code].match(DOMESTIC_ABUSE_CODE_REGEXP) && p[:client_involvement_type] != "A" && p[:delegated_functions_used].to_s == "true" }
+  def not_child_subject_for_this_proceeding(proceeding)
+    match = @proceedings.find { |e| e[:ccms_code] == proceeding.ccms_code && e[:client_involvement_type] != "W" }
```

Comment:
Nit: client_involvement_type's ccms_code of a single character (`W`) is not clear/descriptive to someone reviewing this section of code for the first time.
Possible future improvement?